src/Console/Command/Ci/CiRunCommand.php

```diff
@@ -131,16 +142,24 @@ public function execute(InputInterface $input, OutputInterface $output): int {
       $data['status'] = 'FAIL';
       $job = $this->ciJobFactory->create($options->getJob());
       $data['version'] = $job->getDrupalCoreVersion();
+      $data['allowedToFail'] = $this->env->get("ORCA_IS_ALLOWED_FAILURE") ?? FALSE;
+      $data['branch'] = $this->env->get("ORCA_SUT_BRANCH") ?? FALSE;
       $job->run($options);
 
     }
     catch (OrcaInvalidArgumentException $e) {
       $output->writeln("Error: {$e->getMessage()}");
+      if ($this->env->get("ORCA_IS_ALLOWED_FAILURE")) {
+        return StatusCodeEnum::OK;
+      }
       return StatusCodeEnum::ERROR;
     }
     catch (\Exception $e) {
       $event = new CiEvent($data);
       $this->eventDispatcher->dispatch($event, CiEvent::NAME);
+      if ($this->env->get("ORCA_IS_ALLOWED_FAILURE")) {
```

Comment:
@TravisCarden : Do you think this would be problematic?

Reply:
I think this is the same as the other one: https://github.com/acquia/orca/pull/438/files#r1388318264.

Reply:
Okay